src/imageTile.js

```diff
@@ -1,117 +1,114 @@
 var inherit = require('./inherit');
 var tile = require('./tile');
 
-module.exports = (function () {
-  'use strict';
+/**
+ * This class defines a tile that is part of a standard "image pyramid", such
+ * as an open street map tile set.  Every tile is uniquely indexed by a row,
+ * column, and zoom level.  The number of rows/columns at zoom level z is
+ * `2^z`, the number of pixels per tile is configurable.
+ *
+ * By default, this class assumes that images are fetch from the url, but
```

Comment:
FYI typo: images are fetch**ed** from the url.

Reply:
Thanks.